components/member_list_team/index.ts

```diff
@@ -26,7 +26,7 @@ type Props = {
 }
 
 type Actions = {
-    getTeamMembers: (teamId: string) => Promise<{data: {}}>;
+    getTeamMembers: (teamId: string, page?: number, perPage?: number, options?: {}) => Promise<{data: {}}>;
```

Comment:
based on my comment in redux  - you would then import the `GetTeamMembersOpts` type above here and change this to 
```suggestion
    getTeamMembers: (teamId: string, page?: number, perPage?: number, options?: GetTeamMembersOpts) => Promise<{data: {}}>;
```

Reply:
Done! Thanks